assets/sass/style.scss

```diff
@@ -1,7 +1,99 @@
 @import 'reset';
+html {
+  height: 100%;
+}
 body {
   margin: 0;
+  height: 100%;
+  width: 100%;
```

Comment:
Use google font Roboto. Also, adjust the font-size in small screens. 
Please fix the issues on window resize as well.